docs/experience_date_search/index.md

```diff
@@ -45,6 +45,52 @@ You can enable customers to move between the results pages using the `paginate`
 
 You can assign the value of a Liquid variable to `page_size`.
 
+## Available Months
+The experience date search returns `available_months`, an array of `Date` objects, which can be used to construct the month and year filters for searching.
+
+Each `Date` is dated 1st of the month, and there is one for each month in which a product's slots `start_on` dates occur.
+
+E.g. Today is `1st Jan 2024`. You have a product with a 10 day duration, with the following slots (all future or ongoing);
+- `31th Dec 2023`
+- `15th Jan 2024`
+- `20th Jan 2024`
+- `31th Jan 2024`
+- `28th Feb 2024`
+The `available_months` will be `Date`s with the following dates:
```

Comment:
```suggestion

The `available_months` will be `Date`s with the following dates:
```

formatting: needs an empty line here to force a line break

currently:
![Screenshot 2023-08-16 at 09 09 58](https://github.com/easolhq/easolhq.github.io/assets/12126304/6245e173-0213-4d69-96ad-4b397d15575b)

suggested:
![Screenshot 2023-08-16 at 09 10 06](https://github.com/easolhq/easolhq.github.io/assets/12126304/d74a0f6a-472b-4144-b3be-97116a5bf18c)